apps/experiments/models.py

```
@@ -1819,27 +1828,27 @@ def get_absolute_url(self):
+            trigger_type: The type of conversation end event to trigger. Leaving this as None will not trigger events.
         Raises:
-            ValueError: If propagate is True but commit is not.
+            ValueError: If trigger_type is specified but commit is not.
         """
+        from apps.events.tasks import enqueue_static_triggers
+
         self.update_status(SessionStatus.PENDING_REVIEW)
-        if propagate and not commit:
-            raise ValueError("Commit must be True when propagate is True")
+        if trigger_type and not commit:
+            raise ValueError("Commit must be True when trigger_type is specified")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)